### PR TITLE
Fix migration script when using page-file-dir and add test

### PR DESCRIPTION
**Pull request by dometto**
Thanks to @bartkamphorst for reporting and suggesting the fix!

### CODE REVIEW

File: bin/gollum-migrate-tags [hunk 1]
```diff
@@ -81,6 +81,7 @@ begin
     const = setting.to_s.upcase
     Object.const_set(const, value) unless Object.const_defined?(const)
   end
+  wiki_options[:page_file_dir] = setting(:page_file_dir) ? setting(:page_file_dir) : wiki_options[:page_file_dir] # Allow settings :page_file_dir through PAGE_FILE_DIR constant.
```
Comment (bartkamphorst) [inline bin/gollum-migrate-tags:84]: 👍

File: test/test_migrate.rb [hunk 2]
```diff
@@ -35,33 +54,29 @@
...
+    test 'change spaced filenames to hyphenated filenames' do    
+      load_script(hyphenate: true)
     
       f = ::File.new(::File.join(@path, 'Home.textile'), 'r')
       output = f.read
       assert_equal true, output.include?('[[Bilbo-Baggins.md]]')
       assert_equal true, output.include?('[[evil|Mordor/Eye-Of-Sauron.md]]')
     end
+    
+    test 'migration with page file dir' do
```
Comment (bartkamphorst) [inline test/test_migrate.rb:72]: Do you test `--write` anywhere?

Comment (dometto) [reply test/test_migrate.rb:72]: `NO_DRY_RUN` is true for all the tests! See line 29

**Comment by dometto** [comment]
Shall I merge this and cut a bugfix release, or better to wait longer?

File: test/test_migrate.rb (hunk 2 above)
Comment (bartkamphorst) [reply test/test_migrate.rb:72]: Perfect. 👍

**Comment by bartkamphorst** [comment]
> Shall I merge this and cut a bugfix release, or better to wait longer?

Go ahead!

**Pull request merged**